doc/appendix-de-kassensichv/cash-register-integration/cash-register-integration.md

```diff
@@ -3,39 +3,707 @@ slug: /poscreators/middleware-doc/germany/cash-register-integration
+### Characteristics
 
-The regular workflow of the fiskaltrust.SecurityMechanism in the German market for actions running longer than 45s (German max transaction update time delta), defines the steps required for the creation of a receipt as follows:
+| Implicit flow                                                | Explicit flow                                                |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| No open TSE-transactions to be managed                       | Long-lasting TSE-transactions may need to be managed         |
+| Reduced complexity of business-process handling              | Enhanced complexity of business-process handling             |
+| Each sign-request uses two TSE signatures                    | Less TSE-signatures are used                                 |
+| The TSE signing process could potentially lead faster to performance issues | The risk of performance issues for the TSE signing process is lower |
 
-##### Start-Transaction
+Implicit- and explicit flow can be combined, dependent on the actual needs.
 
+### The implicit flow
+
+#### When to use
+
+This is the regular workflow of the fiskaltrust-SecurityMechanism in the German market for actions which should fit for around 90% of all use cases. 
+
+#### How to use
+
+Sign-call with a [ftReceiptCase for implicit flow](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftreceiptcase#type-of-receipt-ftreceiptcase) + ftReceiptCaseFlag `0x0000000100000000`.
+The up-counting transaction number defined in TR-03153 is responded behind the hash-tag in the property 'ftReceiptIdentification' of 'ReceiptResponse', prefixed by "IT".
+
+**For the implicit flow, ADDITIONALLY the start time of the first transaction of the business-action (e.g. start time of the first order) has to be printed on the receipt as the start-time of the action.**
+
+
+![implicit-flow-start-finish-transaction](media/implicit-flow-start-finish-transaction.png)
+
+*Implicit Flow - Start/Finish Transaction (DE - KassenSichV)*
+
+<details>
+  <summary>Background information</summary>
+  <p>
+
+There has to be a "Start-Transaction" and a "Finish-Transaction" executed against the TSE. In order to speed up these two steps into one call to the 'Sign' method, a special 'ReceiptCaseFlag' is used. Each time this is used in combination with a usual 'ReceiptCase', a "Start-Transaction" is done behind the scenes upfront the final call, using the given 'ReceiptCase'.
+
+**Please be aware:**
+
+- Using a unique identifier in 'cbReceiptReference' that was already used with a 'Sign' call with 'ReceiptCase' "Start-Transaction" will end up in an exception.
+- Because the implicit flow triggers a "Start-Transaction" AND a "Finish-Transaction" against the TSE, for each implicit 'Sign' call two TSE-signatures are consumed.
+
+</p>
+</details>
+
+#### Examples
```

Comment:
I'd add a short paragraph about _why_ we have this examples (e.g. what to look for, ...), and a link to https://middleware-samples.docs.fiskaltrust.cloud/ for more